packages/commerce-server/src/record-new-purchase.ts

```diff
@@ -68,10 +74,18 @@ export type PurchaseInfo = {
   stripeProduct: Stripe.Product
 }
 
+export const purchaseTypeSchema = z.union([
+  z.literal(EXISTING_BULK_COUPON),
+  z.literal(NEW_BULK_COUPON),
+  z.literal(NEW_INDIVIDUAL_PURCHASE),
+])
+type PurchaseType = z.infer<typeof purchaseTypeSchema>
```

Comment:
This gets used both here and in `getServerSideProps` of `apps/testingaccessibility`, so I'd be interested in finding a shared place for this to live outside the `commerce-server`.

Related to what I'm saying in this comment: https://github.com/skillrecordings/products/pull/548/files#r1007082578

Reply:
if it's "just a type" the `types` packages definitely makes sense to me